### PR TITLE
Sami/alpha level editor

### PR DESCRIPTION
**Pull request by Sami362002**
## Summary 

Added new features to level editor and polishing.

Seek Functionality
JSON Reading and Writing
Input to change duration of held notes

### CODE REVIEW

File: assets/levels/test.json
Comment (mluo24) [inline assets/levels/test.json:1]: lol this is kind of scuffed but it works so it works

you can use `prettyPrint` to make it look better

File: core/src/edu/cornell/gdiac/temporary/editor/EditorMode.java [hunk 1]
```diff
@@ -341,15 +436,106 @@ public EditorMode(GameCanvas canvas) {
...
+            if (note.getType() == EditorNote.NoteType.SWITCH){
+                n.type = "switch";
+            }
+            n.duration = note.getDuration();
+            l.bandMembers.get(note.getLane()).notes.add(n);
+        }
+        Json json = new Json();
+        try {
+            file = new FileWriter("test.json");
+            file.write(json.toJson(l));
```
Comment (mluo24) [inline core/src/edu/cornell/gdiac/temporary/editor/EditorMode.java:532]: you can replace this here with `prettyPrint`

**Review by mluo24 — APPROVED** [review]
lgtm

**Review by markim21 — APPROVED** [review]
lgtm!

**Pull request merged**